sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/branch/PullRequestInfo.java

```diff
@@ -31,11 +32,13 @@
   private final String id;
   private final String branch;
   private final String base;
+  private final Date analysisDate;
 
-  public PullRequestInfo(String id, String branch, @Nullable String base) {
+  public PullRequestInfo(String id, String branch, @Nullable String base, Date analysisDate) {
```

Comment:
suggestion: use a long as a timestamp. This is the convention on server side at least